integration-tests/src/lib.rs

```diff
@@ -639,6 +641,11 @@ where
             toml! {
                 [ingestion.index_update]
                 method = "danger_wait_for_completion"
+
+                [snippet_extractor]
+                language = "english"
+                chunk_size = 10
+                hard_chunk_size_limit = 10
```

Comment:
Very low _token_ length so that we can have short dump testing texts. Testing the exact algorithm can be done somewhere else.